### PR TITLE
style(site): add support for third level navigation

### PR DESCRIPTION
**Pull request by abarmina**
**Closes:** [#1322](https://github.com/exadel-inc/esl/issues/1322)
**Preview:**
image

### CODE REVIEW

**Review by NastaLeo — APPROVED** [review]

File: site/src/navigation/sidebar/sidebar-seccondary-nav.less [hunk 1]
```diff
@@ -4,19 +4,24 @@ esl-d-sidebar {
...
     padding: 2px 0;
   }
+
   .sidebar-nav-secondary-item {
     display: block;
     position: relative;
-    padding: 10px 35px 10px 70px;
+    padding-block: 10px;
+    padding-inline-end: 35px;
+    padding-inline-start: var(--padding-inline-start);
```
Comment (dshovchko) [inline site/src/navigation/sidebar/sidebar-seccondary-nav.less:17]:
```suggestion
    padding: 10px 35px 10px var(--padding-inline-start, 0);
```

File: site/src/navigation/sidebar/sidebar-seccondary-nav.less [hunk 2]
```diff
@@ -27,7 +32,23 @@ esl-d-sidebar {
...
+
+    &.sidebar-nav-item-container {
+      padding: 0;
+    }
+
+    .sidebar-nav-item-container-trigger {
+      display: block;
+      padding-block: 10px;
+      padding-inline-end: 35px;
+      padding-inline-start: var(--padding-inline-start);
```
Comment (dshovchko) [inline site/src/navigation/sidebar/sidebar-seccondary-nav.less:44]:
```suggestion
      padding: 10px 35px 10px var(--padding-inline-start, 0);
```

**Review by dshovchko — APPROVED** [review]

**Review by fshovchko — APPROVED** [review]

**Review by yadamskaya — APPROVED** [review]

File: site/src/navigation/sidebar/sidebar-seccondary-nav.less (hunk 1 above)
Comment (ala-n) [reply site/src/navigation/sidebar/sidebar-seccondary-nav.less:17]: Actually, that's not the same thing as padding. The shortcut does not transform into the semantic properties Anna used here.

File: site/src/navigation/sidebar/sidebar-seccondary-nav.less (hunk 2 above)
Comment (ala-n) [reply site/src/navigation/sidebar/sidebar-seccondary-nav.less:44]: Same here

**Pull request merged**